vscode/esbuild.js

```diff
@@ -11,9 +11,7 @@ const baseConfig = {
 const scriptConfig = {
   ...baseConfig,
   target: "es2020",
-  format: "esm",
-  entryPoints: ["./src/propertiesView/script.ts"],
-  outfile: "./out/script.js",
```

Comment:
what was this outfile doing earlier? is this intentional to remove

Reply:
Yes, it is intentional. This was part of the propertiesView, since we removed that this script is also removed.